src/executions.ts

```diff
@@ -34,14 +34,15 @@ export const runTestCase = (
         env: {
             DEBUG: 'true',
             CPH: 'true',
+            PATH: process.env.PATH,
         },
     };
 
-    let process: ChildProcessWithoutNullStreams;
+    let testProcess: ChildProcessWithoutNullStreams;
```

Comment:
Why did you change the variable name?

Reply:
I've changed the variable name to import "node:process" module as its default name.